Core.ApplicationServices/Contract/Write/ItContractWriteService.cs

```diff
@@ -143,6 +151,28 @@ private static Maybe<OperationError> UpdateOrganizationUnit(ItContract contract,
             return Maybe<OperationError>.None;
         }
 
+        private Result<ItContract, OperationError> UpdateSystemAssignments(ItContract contract, IEnumerable<Guid> systemUsageUuids)
+        {
+            return _updateAssignmentHelper.UpdateMultiAssignment
```

Comment:
hmm mangler du måske at slette de gamle assignment objekter bagefter? Hvis de har en optional reference til contract så leaker de måske bare i db bagefter. Du kan gøre det ved at trække systems ud før og så Excepte dem bagefter og slette de assignments der er forsvundet (hvis updatemulti går godt)

Reply:
De bliver slettet i DB

Reply:
Sjovt at de gør det men agreement elements f.eks. gør ikke

Reply:
Jeg ved ikke lige med AgreementElements. Men mange af de andre har jo været fordi de skal have en "parent" og høre til, og vi derfor er nød til at slette dem manuelt. ItSystemUsage og ItContract er jo to forskellige entiteter som sagtens kan leve alene